test/integration/base.py

```diff
@@ -97,25 +195,44 @@ def run_dbt(self, args=None):
         if args is None:
             args = ["run"]
 
+        args = ["--strict"] + args
+
         return dbt.handle(args)
 
     def run_sql_file(self, path):
         with open(path, 'r') as f:
-            return self.run_sql(f.read())
+            statements = f.read().split(";")
+            for statement in statements:
+                self.run_sql(statement)
+
+    # horrible hack to support snowflake for right now
+    def transform_sql(self, query):
+        to_return = query
```

Comment:
rather than maintain two of each seed file, I just replaced `BIGSERIAL` with `BIGINT AUTOINCREMENT` hah.... there's definitely a cleaner way & I didn't spend very much time working on this. What do you think?